test/store/actions/onboarding.test.js

```diff
@@ -35,6 +35,7 @@ describe("test onboarding behaviour", () => {
     const store = mockStore(state);
```

Comment:
We should add a test for the new onboarding method i think

Reply:
+1

Reply:
@marcosmartinez7 

[RLP-947](https://jirainfuy.atlassian.net/browse/RLP-947)